README.md

```diff
@@ -1 +1,3 @@
 # printing_quotes
+The chop method will simply knock off the last character of the 
+string, no matter what it is
```

Comment:
An example of the difference between chop and chomp could explain better what this means :)